product-configuration-extension/package.json.liquid

```diff
@@ -0,0 +1,24 @@
+{%- if flavor contains "react" -%}
+{
+  "name": "{{ handle }}",
+  "version": "1.0.0",
+  "license": "UNLICENSED",
+  "dependencies": {
+    "react": "^17.0.0",
+    "@shopify/ui-extensions-react": "unstable"
+  },
+  "devDependencies": {
+    "@types/react": "^17.0.0"
+  }
+}
+{%- else -%}
+{
+  "name": "{{ name }}",
```

Comment:
I'm late to this review, this is wrong, it should be {{ handle }} (same as above)